### PR TITLE
Mr. D did this feature

### PR DESCRIPTION
**Pull request by DevelHub**
Please review.
I did work and now you have to look at the code to make sure I didn't do any BS stuff.
Thanks for taking the time out of your "busy" schedule to help me out with this.

Good luck with my BS 👍

### CODE REVIEW

File: public/display.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+         <thead>
+          <tr>
+            <th></th>
+           <th>Name</th>
+           <th>Link</th>
+         </tr>
+         </thead>
+         <tbody id="tableLinksBody" >
+          <!-- <tr>
+            <td><input type="checkbox" name="id" value = "1" /></td>
```
Comment (clintonola) [inline public/display.html:35]: Remove commented code

File: public/display.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/skeleton/2.0.4/skeleton.css"
+    />
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>DISPLAY</h1>
+    <!-- <p>Hello <%= user.charAt(0).toUpperCase() + user.slice(1)%></p> -->
```
Comment (clintonola) [inline public/display.html:15]: remove comment code

File: public/display.html (hunk 1 above)
Comment (DevelHub) [reply public/display.html:35]: Hey, you put that... pls remove that on your own time!! Thanks

File: public/display.html (hunk 2 above)
Comment (DevelHub) [reply public/display.html:15]: Not doing, but thanks. 👎

**Pull request merged**